301-storage-spaces-direct-md/nestedtemplates/diskSelection.json

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
we just shipped a platform feature that makes this pattern unnecessary - it will drastically simplify the sample - see: 
https://github.com/Azure/azure-quickstart-templates/tree/master/101-vm-windows-copy-datadisks